docs/source/secured_private_asset_transfer_tutorial.md

```diff
@@ -0,0 +1,394 @@
+
+### Transferring the asset
+
+After the two organizations have agreed to the same price, the asset owner can use the transfer function to transfer the asset to the buyer:
+
+- Smart contract access control ensures that the transfer must be initiated by a member of the organization that owns the asset.
+- The transfer function verifies that the asset's private immutable properties passed to the transfer function matches the on chain hash of the asset data in private collection, to ensure that the asset owner is *selling* the same asset that they own.
+- The transfer function uses the hash of the price agreement on the ledger to ensure that both organizations have agreed to the same price.
+- If the transfer conditions are met, the transfer function adds the asset to the implicit private data collection of the buyer, and deletes the asset from the collection of the seller. The transfer also updates the owner in the public ownership record.
+- Because of the endorsement policies of the seller and buyer implicit data collections, and the state based endorsement policy of the public record (requiring the seller to endorse), the transfer needs to be endorsed by peers from both buyer and seller.
+- The state based endorsement policy of the public asset record is updated so that only a peer of the new owner of the asset can update or sell their new asset.
+- The price agreements are also deleted from both the seller and buyer implicit private data collection, and a sales receipt is created in each private data collection.
+
+## Running the private asset transfer smart contract
+
+You can use the Fabric test network to run the private asset transfer smart contract. The test network contains two peer organizations, Org1 and Org1, that operate one peer each. In this tutorial, we will deploy the smart contract to a channel of the test network joined by both organizations. We will first create a asset that is owned by Org1. After the two organizations agree on a asset price, we will transfer the asset from Org1 to Org2.
+
+## Deploy the test network
+
+We need to deploy an instance of the Fabric test network to run the smart contract. Open a command terminal and navigate to test network directory in your local clone of [fabric-samples](https://github.com/hyperledger/fabric-samples). We will operate from the `test-network` directory for the remainder of the tutorial.
+
+```
+cd fabric-samples/test-network
+```
+
+You can then deploy the network with the following command:
+```
+./network.sh up createChannel -c mychannel
+```
+The script will deploy the nodes of the network and create a single channel named `mychannel` with Org1 and Org2 as channel members. We will use this channel to deploy the smart contract and trade our asset.
+
+### Set the environment variables to operate as Org1
+
```

Comment:
You can now use the test network script to deploy the chaincode instead of going through all these steps, though that does not need to be done in this PR.

Reply:
Yes we can change this in a follow-up PRs for several samples.